packages/charts/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -34,6 +34,7 @@ import {
   domainRangeOfDateForAreaLineVerticalBarChart,
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33376&commitId=e3d279e851f66759812cdc4e903ded45ac574224&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Bottom center.chromium.png|2146| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>